Assets/HoloToolkit/SpatialMapping/Scripts/SpatialProcessing/RoomLoader.cs

```diff
@@ -0,0 +1,51 @@
+using HoloToolkit.Unity;
+using UnityEngine;
+using UnityEngine.VR.WSA.Persistence;
```

Comment:
Please make sure this builds for windows standalone as well.
Also, the VR namespace should be XR